lighthouse-core/test/gather/driver-test.js

```diff
@@ -512,6 +512,7 @@ describe('.gotoURL', () => {
     }
     const replayConnection = new ReplayConnection();
 
+    // @ts-ignore: Coerce to TestDriver for looser typing in function parameters.
```

Comment:
since this is a real driver, it seems a little weird to cast to a fake driver type :)

One option: could ts-ignore an assertion on `loadOptions` instead
```
const loadOptions = {
  waitForLoad: true,
  /** @type {LH.Gatherer.PassContext} */
  // @ts-ignore - we don't need the entire context for the test.
  passContext: {
    passConfig: {
      pauseAfterLoadMs: 0,
      networkQuietThresholdMs: 0,
      cpuQuietThresholdMs: 0,
    },
  },
};
```
(though we should probably eventually have a `passContext` generator if we're really getting all real in this file)